### DISCUSSION TITLE
Alpine Expression Error: $wire.delet is not a function

### DISCUSSION THREAD

**Discussion by kurniagungk** [livewire/livewire#6876] [7. Bugs (archived)]
### Livewire version

3.0.5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

                               @if ($delet == $m->id)
                                            <button type="button" class="btn icon btn-danger"
                                                x-on:click="$wire.delet({{ $m->id }})"><i
                                                    class="bi bi-check"></i></button>
                                        @else
                                            <button type="button" class="btn icon btn-warning"
                                                wire:click="confirm('{{ $m->id }}')"><i
                                                    class="bi bi-trash"></i></button>
                                        @endif

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@kurniagungk what does your Livewire component class look like? And is that your whole component view?
